content/docs/0.9.x/monitoring/dynatrace/install/index.md

```diff
@@ -1,90 +1,110 @@
 ---
 title: Install
-description: Setup Dynatrace monitoring and Keptn integration
+description: Set up Dynatrace monitoring and Keptn integration
 weight: 1
 icon: setup
 ---
 
-## Deploy Dynatrace Operator on Kubernetes
+## Deploy Dynatrace OneAgent Operator on Kubernetes
```

Comment:
Not sure cause the official name is Dynatrace Operator, while the deprecated/old version was called Dynatrace OneAgent Operator. I would use Dynatrace Operator
![image](https://user-images.githubusercontent.com/89777493/135034350-9415c1c0-e552-4ba2-8b5d-20b4e0315186.png)

Reply:
Updated to Dynatrace Operator